src/firewall/functions.py

```diff
@@ -670,3 +670,15 @@ def wrong_args_for_callable(fcn, *a, **kw):
     except TypeError:
         return False
     return True
+
+
+# Dummy class for EOL singleton instance. It's a class, so we
+# can overwrite __repr__().
+class _EOLType:
+    def __repr__(self):
+        # The string representation is the full name of the instance.
+        return "firewall.functions.EOL"
+
+
+# A EndOfLine token, used by tokenizer.
+EOL = _EOLType()
```

Comment:
I'd rather not. `firewall.functions.EOL` is a general thing. It's a token that indicates EOL, it can be used for any kind of parsing.

For good reasons, `firewall.functions` provides serveral API which is only used at one place:
- `firewall.functions.stripNonPrintableCharacters` only in `rich.py`
- `firewall.functions.enable_ip_forwarding` only in `fw_policy.py`.
- `firewall.functions.firewalld_is_active` only in `firewalld.in`
- `firewall.functions.get_nf_conntrack_short_name` only in `fw_policy.py`

Whether something is in `firewall.functions` should depend on whether it's a self-contained concept, that makes sense on it's own (independent from the actual users).

For example, `EOL` is so trival that it doesn't have any tests. But if it were not as trivial, we should add tests for it. And for testing it, `rich.py` is not relevant.

Reply:
I still disagree. We'll never use this for other things. If in the future we do, then sure; lets move it to a common area.

Reply:
moved. It's now `firewall.core.rich.EOL`.